packages/web-config-server/src/apiV1/dataBuilders/DataBuilder.js

```diff
@@ -168,4 +181,21 @@ export class DataBuilder {
   };
 
   areEventResults = results => !!(results[0] && results[0].event);
+
+  eventsDataValuesEntityIdToName = async (models, events, dataElementCodes) => {
```

Comment:
```suggestion
  translateEventEntityIdsToNames = async (models, events, dataElementCodes) => {
```

Also, this function doesn't use `this`, and its logic is independent from this class, so we should make it an independent utility defined in another file and used in `DataBuilder`. We can place on its own file in `@tupaia/web-config-server/src/apiV1/dataBuilderes/helpers`

Reply:
Thanks Kostas, I agree, it looks better. Last week I did the same but was having babel importing error when moving it to `helpers`, couldn't solve it but today it works fine.

Reply:
The code works fine but It actually fail the test, see [codeship](https://app.codeship.com/projects/70159bc0-0dac-0138-fdcb-260b82737f4e/builds/72f51d32-1997-4782-ab92-08f23e441e8b?component=step_validate-and-test_test_test-batch-1_test-web-config-server) . It pass the test when this function is under `dataBuilders` folder, perhaps that's the reason `transform.js` is there.

Reply:
True, although the error seems to be related to `analyticsPerPeriod`?

Also, there are lots of other helpers defined in this folder and used across data builders, so I can't see why this wouldn't work. Could you please investigate?